storybookjs/storybook
# @storybook/addon-cssresources - Hide Code

Issue:

As a user, I want to hide the style of certain CSS Resources as they are quite long and makes switching between them cumbersome.

## What I did

Enable the consumer to hide the CSS Resources' code through the `hideCode` property.

## How to test

- Is this testable with Jest or Chromatic screenshots? Should be
- Does this need a new example in the kitchen sink apps? Don't think so
- Does this need an update to the documentation? Perhaps

If your answer is yes to any of these, please make sure to include it in your PR.
